protocol/credentials.go

```diff
@@ -130,3 +133,27 @@ type CredentialStatusUpdateMessageBody struct {
 	ID     string `json:"id"`
 	Reason string `json:"reason"`
 }
+
+// CredentialRefreshMessage represent Iden3message for credential refresh message
+type CredentialRefreshMessage struct {
+	ID       string                    `json:"id"`
+	Typ      iden3comm.MediaType       `json:"typ,omitempty"`
+	Type     iden3comm.ProtocolMessage `json:"type"`
+	ThreadID string                    `json:"thid,omitempty"`
+
+	Body CredentialRefreshMessageBody `json:"body,omitempty"`
+
+	From string `json:"from,omitempty"`
+	To   string `json:"to,omitempty"`
+}
+
+// CredentialRefreshMessageBody is msg body for refresh message
+type CredentialRefreshMessageBody struct {
+	Refresh []Refresh `json:"refresh"`
```

Comment:
let's call it Credentials `json:"credentials"` as for Offer
https://github.com/iden3/iden3comm/blob/68b9ee93c063731001fb9696676d728f48f1adb7/protocol/credentials.go#L64